editor/src/components/custom-code/code-file.ts

```diff
@@ -183,7 +183,9 @@ export function incorporateBuildResult(
   // Mutates nodeModules.
   fastForEach(Object.keys(buildResult), (moduleKey) => {
     const modulesFile = buildResult[moduleKey]
-    if (modulesFile.transpiledCode != null) {
+    if (modulesFile.transpiledCode == null) {
```

Comment:
Is there ever a case where the build result will only include updated files? I know we originally built the code around supporting incremental builds but don't recall if we ever had that working...

Reply:
Only include updated files? There's a `full-build` case in `generateCodeResultCache` if that's what you mean?

Reply:
Ahhh ok sorry I misunderstood where this code was being called